bin/www

```diff
@@ -4,22 +4,25 @@
  * Module dependencies.
  */
 
-var app = require('../app');
-var debug = require('debug')('braintree_express_example:server');
-var http = require('http');
+import http from 'http';
+import { debug as logger } from 'debug';
+import app from '../app';
+
+const debug = logger('braintree_example:server');
```

Comment:
That is precisely what we're doing. The reasoning is that I want the logging function we ultimately use to be called `debug`. The problem is, the function we'll be using is the return value of its initializer. Since we can't call a function at the same time we import it in ES modules the way we would with CommonJS modules (i.e. `const foo = require('bar')('baz')`), we alias it until we're able to call the initializer.

(sorry if that was either unclear or unintentionally splain-y.)

Reply:
That is precisely what we're doing. The reasoning is that I want the logging function we ultimately use to be called `debug`. The problem is, the function we'll be using is the return value of its initializer. Since we can't call a function at the same time we import it in ES modules the way we would with CommonJS modules (i.e. `const foo = require('bar')('baz')`), we alias it until we're able to call the initializer.

(sorry if that was either unclear or unintentionally splain-y.) (also i wrote this at like 9am but apparently never hit "submit", so sorry about that too.)